content/md/products-super-power/work-on-your-assets.md

```diff
@@ -72,6 +72,23 @@ You also have the completeness indicator directly on the asset thumbnail in the
 
 The completeness is **automatically calculated** as soon as we enrich a required attribute. It means that the bullet point will disappear as soon as the required attribute is filled, and same for the completeness percentage evolution.
 
+# Edit assets
+## Bulk edit a selection of asset attributes
+It is possible to bulk edit your own selection of asset attributes in an asset family.
```

Comment:
Je pense qu'on pourrait rajouter une phrase d'intro en donnant un exemple de use-case. Du genre : 
Let's imagine that you need to add the same tag to several assets, for instance, their main color. Instead of manually adding the tag for each of these assets, you can easily do it once for your selection of assets, via a bulk action. Let's see how to: